README.md

```diff
@@ -181,13 +181,15 @@ For efficient searches we recommend to install `pt` ([the platinum searcher][]).
    fork Spacemacs safely use the `develop` branch where you handle the update
    manually.
 
-3. Launch Emacs. Spacemacs will automatically install the packages it requires.
+3. Launch Emacs. It will automatically install the packages it requires.
 
 4. Restart Emacs to complete the installation.
 
-If the mode-line turns red then be sure to visit the
+   If the mode-line turns red then be sure to visit the
 [troubleshooting][troubleshoot] guide and consult the [FAQ](#faq).
 
+5. Start customizing Spacemacs by reading the [Quick Start Guide](https://github.com/syl20bnr/spacemacs/blob/develop/doc/QUICK_START.org).
```

Comment:
Please use a relative link here. Otherwise this will always link to the develop version.

`[Quick Start guide][QUICK_START.org]` should work in `README.md` since the link is defined at the bottom of the file